src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DefaultRazorConfigurationService.cs

```diff
@@ -50,31 +47,24 @@ public async override Task<RazorLSPOptions> GetLatestOptionsAsync(CancellationTo
                         {
                             Section = "html"
                         },
+                        new ConfigurationItem()
+                        {
+                            Section = "editor"
```

Comment:
Hmmm, since this today is specialcasing vs, lets make the name a little more special cased too to ensure we don't overload settings in VSCode.

```suggestion
                            Section = "vs.editor.razor"
```